direct/nn/rim/tests/test_rim.py

```diff
@@ -90,3 +91,83 @@ def test_rim(
     out = model(img, kspace, mask, sens)[0][-1]
 
     assert list(out.shape) == [shape[0]] + [2] + shape[2:]
+
+
+@pytest.mark.parametrize(
+    "shape",
+    [
+        [3, 3, 16, 16],
+        [2, 5, 16, 32],
+    ],
+)
+@pytest.mark.parametrize(
+    "hidden_channels",
+    [4, 8],
+)
+@pytest.mark.parametrize(
+    "length",
+    [3],
+)
+@pytest.mark.parametrize(
+    "depth",
+    [1, 2],
+)
+@pytest.mark.parametrize(
+    "no_parameter_sharing",
+    [True, False],
+)
+@pytest.mark.parametrize(
+    "instance_norm",
+    [True, False],
+)
+@pytest.mark.parametrize(
+    "dense_connect",
+    [True, False],
+)
+@pytest.mark.parametrize(
+    "skip_connections",
+    [True],  # False
+)
+@pytest.mark.parametrize(
+    "image_init",
+    [
+        "zero-filled",
+        "sense",
+        "input-kspace",
+    ],
+)
```

Comment:
please avoid code duplication. Just skip OpenVINO testing for unsupported parameters

Reply:
But we should do only one check in each test